src/main.rs

```diff
@@ -0,0 +1,116 @@
+use std::env;
+use uuid::Uuid;
+use openssl::error::ErrorStack;
+use openssl::ssl::{SslConnector, SslMethod};
+use postgres::{error::SqlState, Client, Error, Transaction};
+use postgres_openssl::MakeTlsConnector;
+
+// BEGIN ssl_config
+fn ssl_config() -> Result<MakeTlsConnector, ErrorStack> {
+    let builder = SslConnector::builder(SslMethod::tls())?;
+    Ok(MakeTlsConnector::new(builder.build()))
+}
+// END ssl_config
+
+/// Runs op inside a transaction and retries it as needed.
+/// On non-retryable failures, the transaction is aborted and
+/// rolled back; on success, the transaction is committed.
+// BEGIN execute_txn
```

Comment:
I think that putting a `//` comment after the `///` comment breaks docs generation. The `///` is expected to be on the line of code directly before the function definition.

Reply:
Done.